docs/articles/remoting/transports.md

```diff
@@ -77,7 +77,7 @@ akka{
 You'd define a custom HOCON section (`akka.remote.google-quic`) and let Akka.Remote know that it should read that section for a transport definition inside `akka.remote.enabled-transports`.
 
 > [!NOTE]
-> To implement a custom transport yourself, you need to implement the [`Akka.Remote.Transport.Transport` abstract class](http://api.getakka.net/docs/stable/html/F60EF1D2.htm).
+> To implement a custom transport yourself, you need to implement the [`Akka.Remote.Transport.Transport` abstract class](/api/Akka.Remote.Transport.Transport.html).
```

Comment:
Im not linking to the document here, im linking to the api reference. If docfx has a syntax for that........  couldn't find it in their docs.